src/modules/simulator/simulator_params.c

```diff
@@ -51,3 +51,15 @@
  * @group SITL
  */
 PARAM_DEFINE_FLOAT(SIM_BAT_DRAIN, 60);
+
+/**
+ * Simulator Battery minimal percentage
```

Comment:
Can you extend the description for when and how to change this?

Reply:
Done :+1: